qcengine/procedures/geometric.py

```diff
@@ -27,6 +27,9 @@ def compute(self, input_data: 'OptimizationInput', config: 'JobConfig') -> 'Opti
 
         geometric_input = input_data.dict()
 
+        # Temporary patch for geomeTRIC
```

Comment:
It looks like geometric PR #96 is merged. Is this code still necessary?

Reply:
This compatibility code is required for backwards compatibility with geometric, unless we pin a higher geometric version which isn't really required at the moment.